src/utils/tar.zig

```diff
@@ -78,8 +78,12 @@ pub fn parallelUntarToFileSystem(
 
     var pool =
         try HomogeneousThreadPool(UnTarTask).init(allocator, @intCast(n_threads), n_threads);
-    defer pool.deinit(allocator);
-
+    defer {
+        if (!pool.joinForDeinit(.{ .ns = std.time.ns_per_s })) {
```

Comment:
There's a cleaner api for Duration

```suggestion
        if (!pool.joinForDeinit(.fromSecs(1))) {
```